### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ndegwamartin**

### CODE REVIEW

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5870080&bid=19214178)

File: README.md [hunk 1]
```diff
@@ -1,4 +1,55 @@
...
+[Android client unit tests](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/65570428/OpenSRP+Client)
+
+## Deployment
+[Production releases](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/1141866503/How+to+create+a+release+APK)
+
+## Features
+-  Child health care
+-  Antenatal care (ANC)
+-  Postnatal care (PNC) 
+-  All Families register
```
Comment (keymane) [inline README.md:33]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5870080)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,55 @@
...
+### Running the tests
+
+[Android client unit tests](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/65570428/OpenSRP+Client)
+
+## Deployment
+[Production releases](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/1141866503/How+to+create+a+release+APK)
+
+## Features
+-  Child health care
+-  Antenatal care (ANC)
```
Comment (keymane) [inline README.md:31]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5870080)

File: README.md [hunk 3]
```diff
@@ -1,4 +1,55 @@
...
+-  Postnatal care (PNC) 
+-  All Families register
+-  Peer to peer sync
```
Comment (keymane) [inline README.md:34]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5870080)

File: README.md [hunk 4]
```diff
@@ -1,4 +1,55 @@
...
+
+[Android client unit tests](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/65570428/OpenSRP+Client)
+
+## Deployment
+[Production releases](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/1141866503/How+to+create+a+release+APK)
+
+## Features
+-  Child health care
+-  Antenatal care (ANC)
+-  Postnatal care (PNC)
```
Comment (keymane) [inline README.md:32]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5870080)

File: README.md [hunk 5]
```diff
@@ -1,4 +1,55 @@
...
+
+### Running the tests
+
+[Android client unit tests](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/65570428/OpenSRP+Client)
+
+## Deployment
+[Production releases](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/1141866503/How+to+create+a+release+APK)
+
+## Features
+-  Child health care
```
Comment (keymane) [inline README.md:30]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5870080)

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 9
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5870080&bid=19214178)

File: README.md [hunk 6]
```diff
@@ -1,4 +1,57 @@
...
+-  Antenatal care (ANC)
+-  Postnatal care (PNC) 
+-  Malaria module
+-  Stock Management
+-  Service Activity
+-  Family Planning
+-  All Families register
+-  Peer to peer sync
```
Comment (keymane) [inline README.md:36]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5870080)

File: README.md [hunk 7]
```diff
@@ -1,4 +1,57 @@
...
+## Deployment
+[Production releases](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/1141866503/How+to+create+a+release+APK)
+
+## Features
+-  Child health care
+-  Antenatal care (ANC)
+-  Postnatal care (PNC) 
+-  Malaria module
+-  Stock Management
+-  Service Activity
```
Comment (keymane) [inline README.md:33]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5870080)

File: README.md [hunk 8]
```diff
@@ -1,4 +1,57 @@
...
+-  Family Planning
+-  All Families register
```
Comment (keymane) [inline README.md:35]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5870080)

File: README.md [hunk 9]
```diff
@@ -1,4 +1,57 @@
...
+
+[Android client unit tests](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/65570428/OpenSRP+Client)
+
+## Deployment
+[Production releases](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/1141866503/How+to+create+a+release+APK)
+
+## Features
+-  Child health care
+-  Antenatal care (ANC)
+-  Postnatal care (PNC)
```
Comment (keymane) [inline README.md:30]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5870080)

File: README.md [hunk 10]
```diff
@@ -1,4 +1,57 @@
...
+
+## Deployment
+[Production releases](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/1141866503/How+to+create+a+release+APK)
+
+## Features
+-  Child health care
+-  Antenatal care (ANC)
+-  Postnatal care (PNC) 
+-  Malaria module
+-  Stock Management
```
Comment (keymane) [inline README.md:32]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5870080)

File: README.md [hunk 11]
```diff
@@ -1,4 +1,57 @@
...
+### Running the tests
+
+[Android client unit tests](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/65570428/OpenSRP+Client)
+
+## Deployment
+[Production releases](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/1141866503/How+to+create+a+release+APK)
+
+## Features
+-  Child health care
+-  Antenatal care (ANC)
```
Comment (keymane) [inline README.md:29]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5870080)

File: README.md [hunk 12]
```diff
@@ -1,4 +1,57 @@
...
+-  Postnatal care (PNC) 
+-  Malaria module
```
Comment (keymane) [inline README.md:31]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5870080)

File: README.md [hunk 13]
```diff
@@ -1,4 +1,57 @@
...
+-  Stock Management
+-  Service Activity
+-  Family Planning
```
Comment (keymane) [inline README.md:34]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5870080)

File: README.md [hunk 14]
```diff
@@ -1,4 +1,57 @@
...
+
+### Running the tests
+
+[Android client unit tests](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/65570428/OpenSRP+Client)
+
+## Deployment
+[Production releases](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/1141866503/How+to+create+a+release+APK)
+
+## Features
+-  Child health care
```
Comment (keymane) [inline README.md:28]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5870080)

**Comment by ndegwamartin** [comment]
Fixes [#1217 ](https://github.com/OpenSRP/opensrp-client-chw/issues/1217l)

**Pull request merged**